RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -8,16 +8,218 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    private enum Constant {
+        static let avatarImageName = "Face"
+        static let name = "Dmitry"
+        static let avatarFrameWidthHeight = 70
+        static let appleIDText = "Apple ID, iCloud+, Media & Purchases"
+        static let badgesWidthHeight = 30
+        static let notifyXOffset = 120
+        static let notifyCountName = "1"
+        static let updateTextName = "Предложения Apple ID"
+        static let wifiName = ["free_wifi", "homeWIFI"]
+        static let onOFFParam = ["Вкл.", "Выкл."]
+        static let accountNameOffset = -55
+        static let cornerRadius = 5
+        static let newiOSText = "iOS 16.2: уже доступно"
+        static let airplaneImageName = "airplane"
+        static let airModeName = "Авиарежим"
+        static let wifiImageName = "wifi"
+        static let wifiPickerTextName = "Wi-Fi"
+        static let bluetoothImageName = "Bluetooth"
+        static let bluetoothPickerTextName = "Bluetooth"
+        static let antennaImageName = "antenna.radiowaves.left.and.right"
+        static let cellularPickerTextName = "Сотовая связь"
+        static let hotspotImageName = "personalhotspot"
+        static let hotspotModePickerText = "Режим модема"
+        static let airTagImageName = "airtag.radiowaves.forward.fill"
+        static let vpnModeName = "VPN"
+        static let bluetoothImageFrameParam = 25
+        static let NavigationTitle = "Settings"
+
+    }
+    
+    @State var appleIDIsOn = false
+    @State var flightIsOn = false
+    @State var VPNIsOn = false
+    @State var wifiSection = 0
+    @State var bluetoothSection = 0
+    @State var cellularSection = 0
+    @State var modemSection = 0
+    
+    var wifiParam = Constant.wifiName
+    var onOfParam = Constant.onOFFParam
+    
+    
     var body: some View {
-        VStack {
-            Text("")
+        NavigationView {
+            Form {
+                Section {
+                    HStack(spacing: 15) {
+                        Image(Constant.avatarImageName)
+                            .resizable()
+                            .scaledToFill()
+                            .frame(width: CGFloat(Constant.avatarFrameWidthHeight),
+                                   height: CGFloat(Constant.avatarFrameWidthHeight))
+                            .clipShape(Circle())
+                        
+                        VStack(alignment: .leading, spacing: 0) {
```

Comment:
```suggestion
                        VStack(alignment: .leading, spacing: 0) {
```
а нужен ли spacing если он 0?